compliance_checker/cf/cf_18.py

```diff
@@ -2,11 +2,17 @@
 from compliance_checker.base import BaseCheck, TestCtx, Result
 from compliance_checker import MemoizedDataset
 from compliance_checker.cf.cf import CF1_7Check
+from cf.util import reference_attr_variables, string_from_var_type
 from netCDF4 import Dataset
 from compliance_checker.base import BaseCheck, BaseNCCheck, Result, TestCtx
+import requests
+from lxml import etree
+import pyworms
```

Comment:
I'm not sure whether to use `pyworms` or stick to making REST API calls against the WoRMS services directly with `requests`.  `pyworms` does use `requests` under the hood and it has some basic caching functions.  `pyworms` uses the `lru_cache` decorator, but we could in practice cache HTTP requests with a package such as `requests-cache`.